src/error/views/ErrorView.tsx

```diff
@@ -0,0 +1,48 @@
+import { History, Location } from 'history';
```

Comment:
Deze ErrorView component past imo beter in `shared` want ik denk niet dat je super veel error-related files zal hebben, toch?

Lijkt me trouwens ook meer een component dan een view aangezien het soms maar een klein deel van de pagina opneemt.